package exeptions.S_2;

public class t_1 {
    public static void main(String[] args) {
        while (true) {
            try {
                heep.number();
                break;
            } catch (Exception e) {
                System.out.println(" Неверное значение ");
            }
        }
    }
}
